.github/workflows/poc-integration.yml

```diff
@@ -44,14 +44,14 @@ jobs:
 
     - name: Build & Start containers
       run: |
-        docker-compose -f docker-compose.yml -f docker-compose.otel.yml -f docker-compose.pxe.yml down
+        docker-compose -f docker-compose.yml -f docker-compose.dpu.yml -f docker-compose.otel.yml -f docker-compose.pxe.yml down
```

Comment:
The `docker-compose.dpu.yml` gives me pause, but I admit it's better than `docker-compose.xpu.yml` for now. Perhaps we have a `docker-compose.ipu.yml` soon as well.

Reply:
I'm tired of IPU/DPU/XPU naming battles